src/components/MenuButton/MenuButton.tsx

```diff
@@ -0,0 +1,89 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import autobind from 'core-decorators/lib/autobind';
+import {
+  IContextualMenuProps,
+  IContextualMenuItem,
+  IContextualMenuItemProps,
+  ContextualMenuItemType,
+} from 'office-ui-fabric-react/lib/ContextualMenu';
+import { join } from '../../util/classNames';
+import { BaseComponentProps } from '../../util/BaseComponent/props';
+import { IconButton } from 'office-ui-fabric-react/lib/Button';
+import { IconSize, BaseIcon } from '../Icon';
+import MenuButtonItem from './MenuButtonItem';
+import { MenuItemType } from './enums';
+import './MenuButton.css';
+
+export { MenuItemType } from './enums';
+
+const MoreIcon = require('../Icon/icons/More').default;
+
+const renderEmptyIcon = () => null;
+
+export interface MenuButtonItem {
+  key: string;
```

Comment:
We should add comments before each item in the `MenuButtonItem` and `MenuButtonProps` interfaces, this will show up in the editor as TypeScript intellisense descriptions